pyspark/bigdl/models/rnn/README.md

```diff
@@ -103,6 +103,31 @@ PYSPARK_DRIVER_PYTHON=./venv/bin/python PYSPARK_PYTHON=./venv.zip/venv/bin/pytho
 * `--dampening` dampening for momentum, the default value is 0.
 * `--maxEpoch` max number of epochs to train, the default value is 30.
 
+#####In order to use MKL-DNN as the backend, you should:
+1. Define a model with Model(graph container)
+2. Specify the input and output formats of it.
+   For example:
+   ```
+   theDefinedModel.set_input_formats([theInputFormatIndex])
+   theDefinedModel.set_output_formats([theOutputFormatIndex])
+   ```
+   BigDL needs these format information to build IRGraph from StaticGraph for MklDnn computing.
+   
+   The format index of input or output format can be checked
+   in: 
+   ```
+   ${BigDL-core}/native-dnn/src/main/java/com/intel/analytics/bigdl/mkl/Memory.java
+   
+   For instance:
+   public static final int nchw = 7;
```

Comment:
not suitable, you should use the right format

Reply:
Changed.